src/components/molecules/EditSpace/EditSpace.tsx

```diff
@@ -0,0 +1,221 @@
+              <Form.Label>Room type</Form.Label>
+              <Form.Control
+                type="text"
+                ref={register}
+                name="template"
+                value={room.template}
+                placeholder="Room name"
+                custom
+                disabled={true}
+              />
+            </div>
+          </Form.Row>
+
+          <Form.Row>
+            <div className="room-edit-modal__input">
+              <Form.Label>Room name</Form.Label>
+              <Form.Control
+                type="text"
+                ref={register}
+                name="title"
+                placeholder="Room name"
+                custom
+              />
+              {errors.title && (
+                <span className="input-error">{errors.title.message}</span>
+              )}
+            </div>
+          </Form.Row>
+
+          <Form.Row>
+            <div className="room-edit-modal__input">
+              <Form.Label>Room tagline</Form.Label>
+              <Form.Control
+                type="text"
+                ref={register}
+                name="description"
+                placeholder="Description"
+                custom
+              />
+              {errors.description && (
+                <span className="input-error">
+                  {errors.description.message}
+                </span>
+              )}
+            </div>
+          </Form.Row>
+
+          <Form.Row>
+            <Form.Label>Room enabled</Form.Label>
+            <label id={"isEnabled"} className="switch">
+              <input
+                type="checkbox"
+                id={"isEnabled"}
+                name={"isEnabled"}
+                checked={values.isEnabled}
+                onChange={changeRoomState}
+                ref={register}
+              />
+              <span className="slider round"></span>
+            </label>
+          </Form.Row>
+
+          <Form.Row>
+            <div className="room-edit-modal__input">
+              <Form.Label>Room url</Form.Label>
+              <Form.Control
+                type="text"
+                ref={register}
+                name="url"
+                placeholder="Room url"
+                custom
+              />
+              {errors.url && (
+                <span className="input-error">{errors.url.message}</span>
+              )}
+            </div>
+          </Form.Row>
+
+          <Form.Row>
+            <div className="room-edit-modal__input">
+              <Form.Label>Room image:</Form.Label>
+              <ImageInput
+                onChange={handleImageChange}
+                name="image"
+                forwardRef={register}
+                small
+                nameWithUnderscore
+                imgUrl={room.image_url}
+              />
+              {errors.image_url && (
+                <span className="input-error">{errors.image_url.message}</span>
+              )}
```

Comment:
ImageInput is styled component and will be removed in the next clean up PR. The changes are too many already and I don't really want to continue in this PR because it's getting quite complex.